service-base/src/main/java/org/eclipse/hono/service/http/AbstractHttpEndpoint.java

```diff
@@ -83,6 +83,13 @@ public abstract class AbstractHttpEndpoint<T extends ServiceConfigProperties> ex
         }
     };
 
+    /**
+     * A function that tries to parse a string into an Optional boolean.
```

Comment:
can you please be a little more specific as to what kind of string will be passed into what kind of result?

Reply:
Done